src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -827,6 +827,7 @@ public function resultWithEmptyIterator(): array
 
         return [
             [$entity, new class() implements \Iterator {
+                #[\ReturnTypeWillChange]
```

Comment:
I think we should always add the corresponding phpdoc to make things crystal clear.

Reply:
Even to an anonymous test dummy like in this case? I've merely added the attribute here because I don't have a `mixed` type on PHP 7.1.